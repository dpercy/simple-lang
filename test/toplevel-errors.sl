
; defval with an error doesn't break other definitions
123
(def x (error "oops"))
456

; but it does necessarily break things that relied on that value
(+ x 1)
