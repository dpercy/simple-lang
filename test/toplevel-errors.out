123
456
(error "x is not defined")
